modules/java/pages/_partials/gs-additional-steps-for-linux.adoc

```diff
@@ -1,10 +1,10 @@
 // inclusion
 
-:ld-library-path: LD_LIBRARY_PATH=<pathToCbl>/support/linux/x86_64/:$LD_LIBRARY_PATH
+:ld-library-path: LD_LIBRARY_PATH=<lib-directory>/support/libc++:<lib-directory>/support/libicu:$LD_LIBRARY_PATH
 
 [#bmkLinuxSharedLibs]
-.Additional Steps for Linux]
-In addition to setting-up your build environment, you also need to make the supplied native libraries (`<pathToCbl/support`) available:
+.Additional Steps for Linux
+In addition to setting-up your build environment, you also need to make the supplied native libraries available:
```

Comment:
I would prefer the word "support" to the word "native" here.  They do happen to be native, so that's ok, but the purpose they serve is that they are necessary support.

Reply:
Fixed